client/js/table-view.js

```diff
@@ -70,6 +72,25 @@ class TableView {
 		}
 		removeChildren(this.sheetBodyEl);
 		this.sheetBodyEl.appendChild(fragment);
+
+		let numbers = [];
+		for (let i = 0; document.getElementById(i + ', ' + 0) !== null; i++) {
+			let colTotal = 0;
+			for (let j = 0; document.getElementById(i + ', ' + j) !== null; j++) {
```

Comment:
Replace string interpolation with template literals everywhere possible in this function.